doc/LIMITS.md

```diff
@@ -233,14 +279,16 @@ const node = await createLibp2p({
      * Any connection with a `remoteAddress` property that has any of these
      * addresses as a prefix will be immediately rejected
      */
-     deny: [
+    deny: [
       '/ip4/132.14.52.64/tcp/3984',
       '/ip4/234.243.64.2',
       '/ip4/34.42',
       // etc
-    ]
-  }
-})
+    ],
+  },
```

Comment:
These comments should follow the same linting rules as the main codebase; comma-dangle is disallowed.

```suggestion
    ]
  }
```

Reply:
So the `doc-check` command which utilizes https://github.com/bbc/typescript-docs-verifier only checks that Typescript code snippets compile, it's not a linter. That could be added in a follow up PR but wasn't the original intention of this.